resource-managers/kubernetes/core/src/main/scala/org/apache/spark/scheduler/cluster/k8s/ExecutorPodsAllocator.scala

```diff
@@ -347,14 +373,10 @@ private[spark] class ExecutorPodsAllocator(
   }
 
   private def requestNewExecutors(
-      expected: Int,
-      running: Int,
+      numExecutorsToAllocate: Int,
       applicationId: String,
       resourceProfileId: Int,
       pvcsInUse: Seq[String]): Unit = {
-    val numExecutorsToAllocate = math.min(expected - running, podAllocationSize)
-    logInfo(s"Going to request $numExecutorsToAllocate executors from Kubernetes for " +
-      s"ResourceProfile Id: $resourceProfileId, target: $expected running: $running.")
```

Comment:
This message had better be here inside `requestNewExecutors`.

Reply:
My reason to change it was to avoid passing more variables for that method. 

So from this

https://github.com/apache/spark/blob/adc512d4e1837713713fefc6f64af3b0c6c8cdc8/resource-managers/kubernetes/core/src/main/scala/org/apache/spark/scheduler/cluster/k8s/ExecutorPodsAllocator.scala#L343-L346

we would need to pass:

- `targetNum`
- `podCountForRpId`
- `sharedSlotFromPendingPods`

and they only needed for the log line and to calculate `numExecutorsToAllocate`. 

With the current solution  `numExecutorsToAllocate` is enough and when we will extend the current logic to consider more limits to use for allocation then `numExecutorsToAllocate` will be still enough.

WDYT?

Reply:
@dongjoon-hyun what is your opinion?